analysis_driver/pipelines/human_variant_calling_gatk4.py

```diff
@@ -180,9 +180,9 @@ def stage(cls, **params):
         steps_required = [annotate_vcf]
 
     # variant filtering with VQSR
-    filter_snps = stage(VQSRFiltrationSNPs, input_vcf=variant_file, previous_stages=steps_required)
-    filter_indels = stage(VQSRFiltrationIndels, input_vcf=variant_file, previous_stages=steps_required)
-    apply_vqsr = stage(ApplyVQSR, input_vcf=variant_file, previous_stages=[filter_indels, filter_snps])
+    # filter_snps = stage(VQSRFiltrationSNPs, input_vcf=variant_file, previous_stages=steps_required)
+    # filter_indels = stage(VQSRFiltrationIndels, input_vcf=variant_file, previous_stages=steps_required)
+    # apply_vqsr = stage(ApplyVQSR, input_vcf=variant_file, previous_stages=[filter_indels, filter_snps])
```

Comment:
Is this still WIP?

Reply:
No, this is final. 
The commented out code is here because we might come back and revert the decision to use the VQSR stage.

Reply:
I think perhaps a little comment would help not look unfinished.